war/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM eclipse-temurin:17.0.4.1_1-jre as builder
```

Comment:
Nit: I would use an ARG for the version, easier to keep up to date with an updatecli manifest.

Reply:
can the arg be after the declaration of the base image? 🤔